### PR TITLE
Wip master

### PR DESCRIPTION
**Pull request by alexanderGugel**
Let's build a note taking app! Thoughts?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,64 @@
 
 ## Overview
 
-As before, we're mostly giving students practice with the concepts that we've
-been driving home. Students should by this point be pretty comfortable with
-modular applications, and we're just about ready to introduce them to flux.
+In this lab we're going to practice modularizing component architectures by
+building a note taking application.
+
+This is how it's going to look like:
```
Comment (thomastuts) [inline README.md]: How it's going to look / What it's going to look like

File: README.md [hunk 2]
```diff
@@ -8,9 +8,64 @@
...
+
+On the right hand side of the app, we can see the so-called "FileView". The
+`<FileView />` component renders a single file.
+
+Our users can edit the file by entering text into the `<textarea />`.
+
+### `<Toolbar />`
+
+On the top, we can see a toolbar. The `<Toolbar />` component contains two
+buttons. When a user click on the "Add note" button, a new file is being added
```
Comment (thomastuts) [inline README.md]: is ~~being~~ added

File: README.md [hunk 3]
```diff
@@ -8,9 +8,64 @@
...
+to the end of the `fileStore`. When a user clicks the "Remove note" button, the
+currently selected file should be deleted from the `fileStore`.
+
+There should always be at least one file in the sidebar. If the user attempts to
```
Comment (thomastuts) [inline README.md:61]: I like this gotcha! Nice way to spice things up a little. 💣

**Review by thomastuts — CHANGES_REQUESTED** [review]
Looks great! Some minor grammar nits, but that's it.

I also noticed the images are missing (the `assets/` directory and its corresponding images are not checked into the repo yet). Could you fix that too?

File: package.json [hunk 4]
```diff
@@ -45,6 +45,7 @@
     "react-addons-test-utils": "^15.3.0"
   },
   "dependencies": {
+    "classnames": "^2.2.5",
```
Comment (thomastuts) [inline package.json:49]: `sinon` is missing here

File: components/Toolbar.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const React = require('react');
+
+class Toolbar extends React.Component {
```
Comment (thomastuts) [inline components/Toolbar.js]: This can be a stateless functional component

File: README.md (hunk 1 above)
Comment (alexanderGugel) [reply README.md]: I always get that one wrong....

**Comment by alexanderGugel** [comment]
@thomastuts Done! Thanks! Merge?

**Comment by thomastuts** [comment]
@alexanderGugel Yep! I'll let you pull the switch 😄

**Pull request merged**